lib/node/fullnode.js

```diff
@@ -361,16 +361,18 @@ class FullNode extends Node {
   }
 
   /**
-   * Add transaction to mempool, broadcast. Silence errors.
+   * Add transaction to mempool, broadcast. If not sendBlocking
+   * Silence errors.
    * @param {TX} tx
    * @returns {Promise}
    */
 
-  async relay(tx) {
+  async relay(tx, sendBlocking) {
     try {
       await this.sendTX(tx);
     } catch (e) {
-      this.error(e);
+      if (sendBlocking)
+        throw e;
```

Comment:
This should maintain the original behavior if sendBlocking is false

Reply:
fixed ...